column_encoder.py

```diff
@@ -16,18 +17,42 @@
 from sklearn.pipeline import Pipeline
 from sklearn.utils import murmurhash3_32, check_random_state
 
-from fastText import load_model
+from fasttext import load_model
 import category_encoders as cat_enc
 from dirty_cat import SimilarityEncoder, TargetEncoder
 from dirty_cat.similarity_encoder import get_kmeans_prototypes
 
 import gamma_poisson_factorization
+from fast_hash import ngram_min_hash
 
 CE_HOME = os.environ.get('CE_HOME')
 sys.path.append(os.path.abspath(os.path.join(
     CE_HOME, 'python', 'categorical_encoding')))
 from get_data import get_data_path
 
+class LRUDict:
```

Comment:
I think that it would be nice to have this in a separate file. To isolate functionality.